base/client.jl

```diff
@@ -419,7 +419,7 @@ end
 global active_repl
 
 # run the requested sort of evaluation loop on stdio
-function run_main_repl(interactive::Bool, quiet::Bool, banner::Symbol, history_file::Bool, color_set::Bool)
+function run_main_repl(interactive::Bool, quiet::Bool, banner::Symbol, history_file::Bool)
```

Comment:
While looking over the code base for color references to update I noticed this unused function argument. I can add a deprecation for this if required but I thought this function was considered internal.

Reply:
That's internal, so you can just do it